Clarify quantity (#3634)
modelica/ModelicaSpecification: chapters/connectors.tex

  \end{nonnormative}
\item\label{connect-set-quantity-rule}\index{quantity@\robustinline{quantity}!connect set rule}
  In a connection set, all variables having non-empty \lstinline!quantity!-attribute must have the same \lstinline!quantity!-attribute.
  \begin{nonnormative}

>   Note that variables with different quantities may be unit-compatible, in particular Work and Torque.

Reviewer: Maybe formulate as an example?
```suggestion
  Note that variables with different quantities may be unit-compatible, as illustrated by work and torque.
```
or
```suggestion
  Note that variables with different quantities may be unit-compatible, and as seen in the example of work and torque, the typical choice of units may even be identical (newton-meter).
```

>   Note that variables with different quantities may be unit-compatible, in particular Work and Torque.

Author: I will take the first one, as work doesn't normally have unit newton-meter, but joule that I would more say is _compatible_ with newton-meter and adding all of that will be too long.
 I haven't heard of any other similar issues, so I think it is more _the_ example than _an_ example - that's why "in particular" makes the most sense to me.

>   Note that variables with different quantities may be unit-compatible, in particular Work and Torque.

Reviewer: Heard about the angles and dimensionless quantities? :)  Then there are also all the quantities building upon work/torque and angle/dimensionless…

>   Note that variables with different quantities may be unit-compatible, in particular work and torque.

Author: ```suggestion
  Note that variables with different quantities may be unit-compatible, and as seen in the example of work and torque.
```

>   Note that variables with different quantities may be unit-compatible, in particular Work and Torque.

Reviewer: This is now addressed.  Resolving.

>   Note that variables with different quantities may be unit-compatible, and as seen in the example of work and torque.

Reviewer: Typo:
```suggestion
  Note that variables with different quantities may be unit-compatible, as seen in the example of work and torque.
```

>   It is also possible to include medium-information in the quantity to detect connections between different liquids.

Author: It is just describing current practice.

If you look at `Modelica.Media.Interfaces.PartialMedium` you see:
```
type MassFlowRate = SI.MassFlowRate (quantity="MassFlowRate." + mediumName,  ...) "Type for mass flow rate with medium specific attributes";
```

>   It is also possible to include medium-information in the quantity to detect connections between different liquids.

Reviewer: I know that this is current practice.  I just wish that this was _the_ systematic way of doing it, because I find it useful to know that this is the combination of `"MassFlowRate"` and a medium.
